### PR TITLE
UI for adding and removing User and share class code

### PR DESCRIPTION
**Pull request by vinay961861**

### CODE REVIEW

File: src/components/userRoles/UserTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import { t } from 'i18next';
+import { USERTYPES } from '../../common/constants';
+import "./UserTabs.css"
+
+interface UserTabsProps {
+  value: string;
```
Comment (bhanugummana) [inline src/components/userRoles/UserTabs.tsx:9]: make value type as USERTYPES and rename the value

**Review by bhanugummana — CHANGES_REQUESTED** [review]
Add sample UI screenshot

**Comment by vinay961861** [comment]
Screenshot_20240417_092117

**Comment by vinay961861** [comment]
Screenshot_20240417_092352

**Comment by vinay961861** [comment]
Screenshot_20240417_092527

**Comment by vinay961861** [comment]
Screenshot_20240422_121034
Screenshot_20240422_121053

**Pull request merged**